test/distributions/test_distributions.py

```diff
@@ -1203,10 +1201,9 @@ def test_binomial_extreme_vals(self):
 
     def test_binomial_vectorized_count(self):
         set_rng_seed(1)  # see Note [Randomized statistical tests]
-        total_count = torch.tensor([[4, 7], [3, 8]])
+        total_count = torch.tensor([[4, 7], [3, 8]], dtype=torch.float64)
```

Comment:
I don't understand why here and below it's float64. Locally for me Binomial and Multinomial sample() returns float32.

Reply:
See this CI log line from the commit before I added that dtyping for float64: https://fburl.com/z6sdhva3

sample() returned float64 on several of the builds from that run.